### DISCUSSION TITLE
Next v12

### DISCUSSION THREAD

**Discussion by nickluger** [blitz-js/blitz#2886] [Ideas] [answered]
Can Blitz support (pretty cool) Next v12 features, or will it in the near future?

https://nextjs.org/blog/next-12

**Comment by beerose** [comment, marked as answer]
It will! There's some internal work we have to do in order to merge Next's changes easily and we need to convert our Babel plugins to Rust. So we estimate a month for Blitz to have Next 12.

**Reply by nickluger** [reply]
Great to hear, you're incredible! 🚀

**Comment by itsdillon** [comment]
Excited to use the middleware features

**Comment by flybayer** [comment]
In my latest podcast episode I talk about all the new Next.js 12 features and how they relate to Blitz.

The Next.js 12 discussion starts at 13:00

https://flightreview.fm/3

**Reply by nickluger** [reply]
Hey, that's great - thanks for sharing that!
